src/core/client/stream/tabs/comments/containers/PostCommentFormContainer.tsx

```diff
@@ -85,13 +101,39 @@ export class PostCommentFormContainer extends Component<Props, State> {
         onSubmit={this.handleOnSubmit}
         onChange={this.handleOnChange}
         initialValues={this.state.initialValues}
+        min={
+          (this.props.settings.charCount.enabled &&
+            this.props.settings.charCount.min) ||
+          null
```

Comment:
How does this handle when `min = 0` which is falsey?

Reply:
Worst case, add a test maybe when `min === 0`?

Reply:
That's fine `0` and `null` is the same in this case.